wfe/wfe.go

```diff
@@ -171,7 +180,17 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h web
 		web.WFEHandlerFunc(func(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
 			// We do not propagate errors here, because (1) they should be
```

Comment:
This is a good opportunity to revisit the "do not propagate errors" decision. I think we should go ahead and return an error if there's a problem getting a nonce. That will help make any issues that arise during deployment more visible to our metrics, and will make the error messages clearer in clients.